xbmc/cores/VideoPlayer/Process/gbm/VideoBufferDRMPRIME.cpp

```diff
@@ -0,0 +1,132 @@
+/*
+ *  Copyright (C) 2017-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "VideoBufferDRMPRIME.h"
+#include "threads/SingleLock.h"
```

Comment:
empty line is missing between own header and other Kodi headers, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
thanks, fixed